### DISCUSSION TITLE
How do I use the VGA Output?

### DISCUSSION THREAD

**Discussion by HoodieCoder05** [hneemann/Digital#1212] [Q&A]
I can't figure out how to get it to work

**Comment by hneemann** [comment]
The file `example/processsor/VGAExample` contains a example with a processor and a simple VGA card.
This example runs on an FPGA board and shows a graphic if a real VGA monitor is connected and it also shows the graphic on the VGA component in the simulation. 
You have to let the simulation run for a while because the VGA component needs two vertical sync pulses to synchronize.
